sdcm/cluster.py

```diff
@@ -300,6 +300,51 @@ def __init__(self, name, parent_cluster, ssh_login_info=None, base_logdir=None,
         self._kernel_version = None
         self._uuid = None
         self.scylla_network_configuration = None
+        self._datacenter_name = None
+        self._node_rack = None
+
+    def _is_node_ready_run_scylla_commands(self) -> bool:
+        """
+        When node is just created and started to configure, during first node initializing, it is impossible to connect to the node yet and
+        `remoter` object is None. So we cannot connect to the node and run there needed commands.
+
+        When `remoter` is ready but cluster is not configured yet, `nodetool status` (and other Scylla commands) cannot be run.
+        Wait when it became available.
+        """
+        # During first cluster remoter is not available and may cause to failure
+        if not self.remoter:
+            self.log.warning("Remoter is not available")
+            return False
+
+        # During first cluster initializing the status info is not available and may cause to failure
+        if not self.db_up():
```

Comment:
This can be long, and it would be holding a log

I think we should just try executing the command we want, and fallback to not printing the data

Reply:
getting nodetool status has retrying and it takes a lot of time also. Checking `db_up` will be performed in the beginning only. By how it runs in the test - it is not a problem.
But I'll think about another way

Reply:
`db_up` just check if the DB is up and does not wait for that. So there is not impact on performance.
If `not db_up` - not print dc and rack names. This is the only impact